plugins/search-backend-module-stack-overflow/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@backstage/plugin-search-backend-module-stack-overflow",
```

```suggestion
  "name": "@backstage/plugin-search-backend-module-stack-overflow-collator",
```

open question on naming

---

Done 🙌🏻 : https://github.com/backstage/backstage/pull/20456/commits/7cc8a4ce0d728c306b8344057e29cfe4753a0fd0